### ISSUE TITLE
"Quick tutorial" link broken in the wiki

### ISSUE THREAD

**Issue by alexprp** [nick-youngblut/CLdb#1]
Hi Nick, Whitaker Lab user here. 

It looks like the "Quick Tutorial" link on the wiki is broken? Clicking on it kicks me back to the main wiki page with an error at the top of the page saying "You do not have permission to update this wiki."

Which is odd because the link is just https://github.com/nick-youngblut/CLdb/wiki/Quick-tutorial so I'm not sure why it's an edit link. 

Does the quick tutorial exist? If so then the general user base does not have access to it.

**Comment by nick-youngblut** [comment]
Yeah, it appears that the "quick tutorial" wiki page is gone. I don't know why. Clicking on https://github.com/nick-youngblut/CLdb/wiki/Quick-tutorial  for me just takes me to a "create new page" dialogue. I'll see if I can figure out why
